README.md

```diff
@@ -144,3 +144,42 @@ The following is the request flow (seen in blue in the architecture diagram abov
     ```
 
 1. You can see the pods with `kubectl get pods.`
+
+## Uninstalling the component
+1. Remove the demo application
+    ```
+    kubectl delete -f test/app/service.yml
+    ```
+2. If you previously set every service to respond to the `Prefer: respond-async` header, remember to remove this setting by updating the ingress.class to null or by updating the ingress.class to the ingress.class you would like to use, for example `kourier`.
+    ```
+    kubectl patch configmap/config-network \
+    -n knative-serving --type merge \
+    -p '{"data":{"ingress.class":null}}'
+    ```
+
+    ```
+    kubectl patch configmap/config-network \
+    -n knative-serving \
+    --type merge \
+    -p '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+    ```
+3. Remove the producer component
+    ```
+    ko delete -f config/async/100-async-producer.yaml
+    ```
+4. Remove the `RedisStreamSource` and tls secret
+   ```
+   kubectl delete -f config/async/100-async-redis-source.yaml
+   kubectl delete -f config/async/tls-secret.yaml
+   ```
+5. Remove the redis setup - make sure you are using this command in the `eventing-redis` project
```

Comment:
"Remove the redis setup" wording is a little weird to me. Can we be more precise with "Remove the redis source" or "remove the eventing-redis component"